src/database/database-postgresql.cpp

```diff
@@ -495,7 +495,7 @@ void PlayerDatabasePostgreSQL::savePlayer(RemotePlayer *player)
 	execPrepared("remove_player_inventories", 1, rmvalues);
 	execPrepared("remove_player_inventory_items", 1, rmvalues);
 
-	std::vector<const InventoryList*> inventory_lists = sao->getInventory()->getLists();
+	const std::vector<InventoryList *> &inventory_lists = sao->getInventory()->getLists();
```

Comment:
```suggestion
	const auto &inventory_lists = sao->getInventory()->getLists();
```
(same in other files)